src/WPUpdatePhp.php

```diff
@@ -27,7 +27,21 @@ public function __construct( $minimum_version, $recommended_version = null ) {
 	 * @param $name string Name of the plugin to be used in admin notices
 	 */
 	public function set_plugin_name( $name ) {
-		$this->plugin_name = $name;
+		if ( empty( $this->plugin_name ) ) {
+			$this->plugin_name = __( 'this plugin' );
+		} else {
+			$this->plugin_name = $name;
+		}
+	}
+
+	/**
+	 * @param $name
+	 *
+	 * @return bool
+	 */
+	public function does_my_plugin_meet_required_php_version( $name ) {
```

Comment:
Would you use this function to do the name set and version check in one call? Just syntactic sugar, right (just making sure I don't miss anything)?